src/views/v2/rsvp/actions/success.php

```diff
@@ -30,6 +37,8 @@ class="tribe-common-form-control-toggle__input tribe-tickets__rsvp-actions-succe
 			type="checkbox"
 			value="toggleOne"
 			data-rsvp-id="<?php echo esc_attr( $rsvp->ID ); ?>"
+			data-attendee-ids="<?php echo esc_attr( $attendee_ids ); ?>"
+			data-opt-in-nonce="<?php echo esc_attr( $opt_in_nonce ); ?>"
```

Comment:
We need to receive some sort of `$checked` from the BE to see if this input should be `checked()`.